### PR TITLE
fix [frontend]:LandingPage.js (1st) added

### PR DESCRIPTION
**Pull request by shrutilenka**
pr for #38 

@varisha-025

### CODE REVIEW

**Comment by varisha-025** [comment]
@shrutilenka I will review it by tomorrow

**Comment by varisha-025** [comment]
but first, stick to the commit conventions as mentioned in the README.md. Rewrite the commit name, do not create another commit for renaming

**Comment by shrutilenka** [comment]
> @shrutilenka but first, stick to the commit conventions as mentioned in the README.md. Rewrite the commit name, do not create another commit for renaming. Also, please use Material UI for this.

sorry I missed it.What can I do for now?
I don't have any experience using material Ui .I did it using tailwind css.

**Comment by varisha-025** [comment]
@shrutilenka I need time for reviewing as I have other things to do. Please wait till tomorrow

**Comment by varisha-025** [comment]
@shrutilenka `import BsCircleFill from 'react-icons/bs';`, this import statement is throwing error

**Comment by shrutilenka** [comment]
@varisha-025  I added the illustrations successfully!

**Comment by varisha-025** [comment]
@shrutilenka  Are you not able to see the requested changes?

**Comment by shrutilenka** [comment]
> @shrutilenka Are you not able to see the requested changes?

3hrs before you asked to change the illustrations. That's what I changed and committed again. Apart from that nothing else has been mentioned for changes.The day before yesterday you had asked to rename the pr and i did change that one too and one thing that i could work on is using materialUI for CSS.I don't have any prior experience so i am not able to work on it.
if anything else i have missed then do let me know.waiting for your response.

**Review by varisha-025 — CHANGES_REQUESTED** [review]

**Comment by varisha-025** [comment]
@shrutilenka I hope they are visible to you now.

**Review by varisha-025 — CHANGES_REQUESTED** [review]

File: client/src/components/Layout/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import {BsCircleFill} from 'react-icons/bs';
```
[the remark this answers is not in the document]
Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:2]: npm install react-icons --save

use this command to install the icon

[the remark this answers is not in the document]
Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:2]: Done. Change the color of the circles to #dabb59

**Comment by shrutilenka** [comment]
@varisha-025 i have changed what you had mentioned and can you help me with how to squash all commits into one?

**Comment by varisha-025** [comment]
@shrutilenka https://www.youtube.com/watch?v=V5KrD7CmO4o&t=44s, This tutorial will help you with squashing all the commits. If you still facce issue, let me know

File: client/src/components/Layout/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                    <div class="flex justify-center">
+                      <div class="rounded-lg shadow-lg bg-black max-w-sm ">
+                        <div class="p-2">
+                          <h5 class="text-white text-xl font-medium mb-2">Add sale info</h5>
+                        </div>
+                        <a href="#!">
+                          <img class="rounded-t-lg p-2" src={img3} alt=""/>
+                        </a>
+                        <div class="p-6">
+                          <p class="text-white text-base mb-4">
```
[the remark this answers is not in the document]
Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js]: Still there are some spelling mistakes like `phone number`

File: client/src/components/Layout/LandingPage.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            <div className='w-[44rem] mx-auto text-center'>
+            <h3 className='text-[2.5rem] font-bold'>Stand out with a free Business Profile</h3>
+            <p className='text-[1.25rem] pt-3 font-thin'>Turn people who find you on this website into new customers with a free Business Profile for your 
+                storefront or service area.Personalise your profile with promotion product photos, offers, posts, and more.</p>
+            </div>
+            <div  className='w-[46rem] mx-auto mt-[4rem] text-center'>
+                <h3 className='text-[2.5rem] font-bold'>First impression is the last impresion</h3>
+                <p  className='text-[1.25rem] pt-3 font-thin'>Highlight essential info and show what makes your business unique, right on your Business Profile.</p>
+            </div>
+            <div className='flex flex-row space-x-[6rem] justify-center pt-[10rem] px-[12rem]'>
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:19]: change `pt-[10rem] px-[12rem]` to `py-[11rem] px-[5rem]` and `space-x-[6rem]` to `space-x-[3rem]`

File: client/src/components/Layout/LandingPage.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                <div className='flex-row space-x-2 inline-flex'>
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:20]: Add `w-[30rem] h-[30rem]`

File: client/src/components/Layout/LandingPage.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+                    <div className="text-[3.5rem] text-[#dabb59]">
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:21]: add `pr-1`

File: client/src/components/Layout/LandingPage.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+                        <div className='relative items-center flex'>
+                         <BsCircleFill  />
+                         <p className='text-white absolute text-[2.5rem] ml-4'>1</p>
+                        </div>
+                    </div>
+                    <div class="flex justify-center">
+                      <div class="rounded-xl shadow-lg bg-[#3d3d3d] max-w-sm ">
+                        <div class="pt-2 px-4">
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:29]: Change `pt-2` to` pt-3`

File: client/src/components/Layout/LandingPage.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+                          <h5 class="text-white text-[1.25rem] font-medium  pt-2">Add essential information</h5>
+                        </div>
+                        <a href="#!">
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:32]: No need of anchor tag

File: client/src/components/Layout/LandingPage.js [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+                          <img class="rounded-t-lg px-6 pt-8" src={img1} alt=""/>
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:33]: Add an alt text

File: client/src/components/Layout/LandingPage.js [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+                        </a>
+                        <div class="p-6">
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:35]: Change `p-6` to `p-4`

File: client/src/components/Layout/LandingPage.js [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+                          <p class="text-white text-[1.25rem] text-base ">
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:36]: change `text-[1.25rem]` to `text-[1.075rem]` and remove `text-base`

File: client/src/components/Layout/LandingPage.js [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+                            Let customers know info like hone number, products you sell, health and safety
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:37]: it should bephone number

File: client/src/components/Layout/LandingPage.js [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+                            measures so customers know what to expect.
+                          </p>
+                        </div>
```
Comment (varisha-025) [inline client/src/components/Layout/LandingPage.js:40]: The above changes should be done for all the three cards

**Review by varisha-025 — CHANGES_REQUESTED** [review]

**Comment by shrutilenka** [comment]
hey, @varisha-025 I will be updating the changes by tomorrow 10 pm. Due to some emergency, I am not able to work on it today.

File: client/src/components/Layout/LandingPage.js (hunk 12 above)
Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:40]: Also make them mobile responsive so inmobile view the size reduces and they are aligned vertically one after each other

File: client/src/components/Layout/LandingPage.js (hunk 4 above)
Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: in which line?

Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: @varisha-025

File: client/src/components/Layout/LandingPage.js (hunk 5 above)
Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:21]: mention the line @varisha-025

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:21]: Line 21

File: client/src/components/Layout/LandingPage.js (hunk 4 above)
Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: Line 19

**Comment by varisha-025** [comment]
@shrutilenka I am hoping the changes will be done till tomorrow as you have not done them today?

File: client/src/components/Layout/LandingPage.js (hunk 4 above)
Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: image



@varisha-025

Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: @varisha-025 once review your required changes pls.

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: Try line 20

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: Maybe because of screen difference as I have 13 inch mac.

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: Shift the first card's image upwards and increase the height by 1 or 2 rem. whatever looks good

Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: okey

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: Do this and make the PR

Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: all the 3 imgs should be in one online otherwise it wont look uniform to user's eye

Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: better i will increase the height

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: Did you make it responsive?

Comment (shrutilenka) [reply client/src/components/Layout/LandingPage.js:20]: No I haven't. You add another issue to make it responsive.My exams have started, so I am not able to devote much time to it

Comment (varisha-025) [reply client/src/components/Layout/LandingPage.js:20]: okay. force push the changes then

**Review by varisha-025 — APPROVED** [review]

**Pull request merged**